src/models/User.model.js

```diff
@@ -1,9 +1,21 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const User = sequelize.define(
-  // your code goes here
+  'User',
+  {
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  },
+  {
+    tableName: 'users',
+    createdAt: false,
+    updatedAt: false,
```

Comment:
The `createdAt` and `updatedAt` fields are set to `false`, meaning they won't be automatically managed by Sequelize. If you want to track creation and update times, consider setting these to `true`.